examples/rmrk/resource/wasm/tests/resources.rs

```diff
@@ -195,6 +199,87 @@ fn adding_non_existing_part_to_resource_fails() {
     assert!(run_result.contains(&(ADMIN_ID, expected_response)));
 }
 
+#[tokio::test]
+async fn adding_resource_to_storage_using_client_succeeds() {
```

Comment:
Can we please renaming it so it matches the other similar methods `adding_resource_to_storage_by_admin_via_client_succeeds` and place it next to those methods so it's seen how the same functionality is tested using different means